### DISCUSSION TITLE
How can I map double-clicking SHIFT like in Intellij?

### DISCUSSION THREAD

**Discussion by TwIStOy** [neovim/neovim#30776] [Q&A]
I tried `vim.on_key` but this callback does not include events for pressing modifier keys. Do I have other options?

**Comment by DMunkei** [comment]
Wouldn't this work similar to any other mapping by assigning it on the left hand side? If you're unsure what the key is called that you need to pass to the mapping.

You can go into insert mode, press `C-v` and then `shift` it should show you the character code. I assume this would allow you to use the shift key by itself in the mapping.

**Comment by zeertzjq** [comment]
You can't

**Comment by stCarolas** [comment]
You can remap double-shift to custom key with this tool:
https://github.com/rvaiya/keyd
